packages/frontend/src/components/SettingsValidator/index.tsx

```diff
@@ -0,0 +1,180 @@
+import { ValidationResponse } from '@lightdash/common';
+import {
+    Alert,
+    Button,
+    clsx,
+    Flex,
+    Group,
+    Paper,
+    Table,
+    Text,
+    useMantineTheme,
+} from '@mantine/core';
+import {
+    IconAlertCircle,
+    IconChartBar,
+    IconCheck,
+    IconLayoutDashboard,
+    IconTable,
+} from '@tabler/icons-react';
+import { FC } from 'react';
+import { useTableStyles } from '../../hooks/styles/useTableStyles';
+import { useTimeAgo } from '../../hooks/useTimeAgo';
+import {
+    useValidation,
+    useValidationMutation,
+} from '../../hooks/validation/useValidation';
+import MantineIcon from '../common/MantineIcon';
+import { IconBox } from '../common/ResourceView/ResourceIcon';
+
+const UpdatedAtAndBy: FC<
+    Required<Pick<ValidationResponse, 'lastUpdatedAt' | 'lastUpdatedBy'>>
+> = ({ lastUpdatedAt, lastUpdatedBy }) => {
+    const timeAgo = useTimeAgo(lastUpdatedAt);
+
+    return (
+        <>
+            <Text fz="xs" fw={500}>
+                {timeAgo}
+            </Text>
+            <Text fz="xs" color="gray.6">
+                by {lastUpdatedBy}
+            </Text>
+        </>
+    );
+};
+
+export const SettingsValidator: FC<{ projectUuid: string }> = ({
+    projectUuid,
+}) => {
+    const { classes } = useTableStyles();
+    const theme = useMantineTheme();
+
+    const { data, isSuccess } = useValidation(projectUuid);
+    const { mutate: validateProject, isLoading: isValidating } =
+        useValidationMutation(projectUuid);
+
+    const Icon = ({
+        validationError,
+    }: {
+        validationError: ValidationResponse;
+    }) => {
+        if (validationError.chartUuid)
+            return <IconBox icon={IconChartBar} color="blue.8" />;
+        if (validationError.dashboardUuid)
+            return <IconBox icon={IconLayoutDashboard} color="green.8" />;
+        return <IconBox icon={IconTable} color="indigo.6" />;
+    };
+
+    // TODO: add error state
+
+    return (
+        <>
+            {isSuccess && (
+                <Paper withBorder sx={{ overflow: 'hidden' }}>
+                    <Group
+                        position="apart"
+                        p="md"
+                        sx={{
+                            borderBottomWidth: 1,
+                            borderBottomStyle: 'solid',
+                            borderBottomColor: theme.colors.gray[3],
+                        }}
+                    >
+                        <Text fw={500} fz="xs" c="gray.6">
+                            {/* TODO: show created at timestamp on hover of formatted date instead? */}
```

Comment:
TODO ?

Reply:
need to check with @TuringLovesDeathMetal , but it's currently matching the design. Shouldn't be non-blocking for merge IMO

Reply:
I think it's fine without it , so no need for the to-do :) (sorry, my designs are like 80% instead of 100% so there's a bit of room for creativity haha)

Reply:
thanks @TuringLovesDeathMetal ! 
good to know haha creativity: through the roof